app_server/app/adapters/socket/RoomsManager.js

```diff
@@ -106,6 +118,10 @@ Class.removeRoom = function(roomName) {
   delete this.rooms[roomName];
 };
 
+Class.getRooms = function() {
+  return this.rooms;
+};
+
```

Comment:
I think it's better to have `getters` in this case since by right, we should return copies of the rooms instances so they won't be incorrectly mutated elsewhere i.e. read-only. So this leaves room for implementation change in the future.